docs/self-managed/platform-architecture/overview.md

```diff
@@ -19,7 +19,11 @@ In this architecture, Camunda Platform 8 Self-Managed can be accessed as follows
 - Keycloak authentication: `https://camunda.example.com/auth`
 - Zeebe gateway: `grpc://zeebe.camunda.example.com`
 
-It's also possible to set up an Ingress for each component. For more details, visit our [combined and separated Ingress set up guide](../platform-deployment/helm-kubernetes/guides/ingress-setup.md).
+It's also possible to set up an Ingress for each component. For more details, visit our [combined and separated Ingress setup guide](../platform-deployment/helm-kubernetes/guides/ingress-setup.md).
+
+:::caution Web Modeler
```

Comment:
If Web Modeler as an app supports working with sub-paths right now, I'd suggest doing this in 1 shot and supporting the combined setup since it's a really small change on the Helm side.

If Web Modeler doesn't support working with sub-paths, then it's fine to do it this way.

Reply:
Unfortunately, Web Modeler itself doesn't support running on a sub-path yet (but we're working on it 🙂). I didn't want to wait with the documentation updates until the feature is ready though. The changes for the Helm chart are already prepared (see this [draft PR]( https://github.com/camunda/camunda-platform-helm/pull/569)).